src/components/KButton/KButton.vue

```diff
@@ -387,15 +387,29 @@ export default defineComponent({
       color: var(--grey-400) !important;
     }
   }
+
   &.btn-link {
     color: var(--KButtonBtnLink, var(--blue-500, color(blue-500)));
     background-color: transparent;
+    padding: 0;
+
     &:hover:not(:disabled) {
       text-decoration: underline;
     }
+
     &:focus {
-      @include boxShadow(var(--KButtonOutlineBorder, var(--blue-500, color(blue-500))), 0, 2px);
+      text-decoration: underline;
+    }
+
+    &:focus-visible,
+    &:focus-visible:disabled {
+      border-radius: unset;
+      text-decoration: none;
+      outline: Highlight auto 1px;
```

Comment:
I don't think this `Highlight` property is valid and/or doing anything.

Just `auto 1px` should be sufficient I believe.

Reply:
But the order is wrong (`Highlight` has to go at the end)... however, if you put it at the end, the color then turns lighter and isn't ideal 😬

Reply:
changed to default to `auto 1px black`. I think that's right?